source/database/create-database.rst

```diff
@@ -35,34 +35,39 @@ In order to launch a new database instance we need to first decide on a few
 options, these include:
 
 * The **datastore type** which defines the type of database to be deployed.
-  For this instance we are using MySQL.
 * The **database version** which is informed by your datastore type.
-* The **flavor**, which determines the vCPU and RAM assigned to the
-  instance.
+* The **flavor**, which determines the vCPU and RAM assigned to the instance.
 * You will also need to source an OpenRC file in the correct region
-
-.. Note::
-  It is also necessary to have an existing network on the project that you
+* It is also necessary to have an **existing network** on the project that you
```

Comment:
The network should include a subnet that åttached to a router.

Reply:
I've made these requirements explicit